pyLSV2/client.py

```diff
@@ -2213,3 +2216,144 @@ def tst_record_data(
         content = self._llcom.telegram(lc.RSP.T_BD)
 
         return recorded_data
+
+    def real_time_readings(
+        self, signal_list: List[ld.ScopeSignal], time_readings: int, intervall_us: int
```

Comment:
This function duplicates most of the code from tst_record_data but it is probably much better suited for the intended purpose so I think tst_record_data should be removed completely, no reason to keep duplicate code.

Reply:
I removed tst_record_data  function.